src/interp/interp.cc

```diff
@@ -1397,6 +1443,10 @@ ValueTypeRep<R> IntEqz(ValueTypeRep<T> v_rep) {
   return ToRep(v_rep == 0);
 }
 
+ValueTypeRep<uint32_t> RefIsNull(ValueTypeRep<Ref> v_rep) {
+  return ToRep(v_rep.index == kInvalidIndex);
```

Comment:
Are null and invalid distinct? Probably not so I think this is fine (in that, uninitialized == null, so there's no other state)

Reply:
For now yes.  We could revisit this later.